torch/jit/annotations.py

```diff
@@ -95,15 +95,15 @@ def parse_type_line(type_line):
     arg_ann_str, ret_ann_str = split_type_line(type_line)
 
     try:
-        arg_ann = eval(arg_ann_str, _eval_env)
+        arg_ann = eval(arg_ann_str, _eval_env)  # noqa: P204
     except (NameError, SyntaxError) as e:
         raise RuntimeError("Failed to parse the argument list of a type annotation: {}".format(str(e)))
 
     if not isinstance(arg_ann, tuple):
         arg_ann = (arg_ann,)
 
     try:
-        ret_ann = eval(ret_ann_str, _eval_env)
+        ret_ann = eval(ret_ann_str, _eval_env)  # noqa: P204
```

Comment:
I installed `flake8-string-format` and don't get an error on this line. Are you sure this `noqa` is needed?

Reply:
I got P204 and didn't dig deeper.

Reply:
I don't think these suppressions are necessary for CI but it seems relatively harmless